expressjs-backend/util/s3.js

```diff
@@ -11,28 +11,30 @@ AWS.config.update({
 const s3 = new AWS.S3();
 
 const uploadImageToS3 = async (imageName, imageData) => {
-    const params = {
-        Bucket: "haggleimgs",
-        Key: imageName,
-        Body: imageData
-      };
-    
-      try {
-        const data = await s3.upload(params).promise();
-        console.log("Image uploaded successfully:", data.Location);
-        return data.Location; // Return the URL of the uploaded image
-      } catch (error) {
-        console.error("Error uploading image:", error);
-        throw error;
-      }
+  if (!imageData || !imageName) {
+    throw new Error("Invalid image data or name");
+  }
+  const params = {
+      Bucket: "haggleimgs",
+      Key: imageName,
+      Body: imageData
+    };
+  
+    try {
+      const data = await s3.upload(params).promise();
+      console.log("Image uploaded successfully:", data.Location);
```

Comment:
remove console.log statements or comment them out once feature is implemented. Up to you if you want to do this. It can be easier for future debugging if terminal isn't bombed.

Reply:
Done